.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,36 +45,25 @@ Need Support?
 ```
 </details>
 
-### Device (Host) Operating System
-<!-- Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core, etc. -->
-
-### Architecture
-<!-- amd64, arm32, etc. -->
-
-### Container Operating System
-<!-- Linux containers, Windows containers -->
+### Device Information
+* Host OS [e.g. Ubuntu 16.04, Ubuntu 18.04, Windows IoT Core]: 
+* Architecture [e.g. amd64, arm32, arm64]: 
+* Container OS [e.g. Linux containers, Windows containers]: 
 
 ### Runtime Versions
+* iotedged [run `iotedge version`]: 
+* Edge Agent [image tag (i.e. 1.0.0)]: 
+* Edge Hub [image tag (i.e. 1.0.0)]: 
+* Docker/Moby [run `docker version`]: 
 
-#### iotedged
-<!-- Run `iotedge version` -->
-
-#### Edge Agent
-<!-- Image tag (i.e. 1.0.0) -->
-
-#### Edge Hub
-<!-- Image tag (i.e. 1.0.0) -->
-
-#### Docker
-
-```
-<Run `docker version` (`docker -H npipe:////./pipe/iotedge_moby_engine version` for Moby on Windows) and paste here>
-```
+Note: on Windows include the host option `docker -H npipe:////./pipe/iotedge_moby_engine version`
 
 ## Logs
-<!-- Please share as many logs as possible. This will help debugging -->
-<!-- Follow [diagnostic steps](https://docs.microsoft.com/en-us/azure/iot-edge/troubleshoot#standard-diagnostic-steps) to help extract useful information. -->
-<!-- Don't forget to remove any connection string information! -->
+<!--
+Please share as many logs as possible. This will help debugging
+Follow [diagnostic steps](https://docs.microsoft.com/en-us/azure/iot-edge/troubleshoot#standard-diagnostic-steps) to help extract useful information.
+Don't forget to remove any connection string information!
+-->
 
 <details>
 <summary>iotedged logs</summary>
```

Comment:
GH won't let me comment on the line, but below where we have `<Paste here>` between triple backticks, can you change it to `<Paste here between the triple backticks>` or something? We get many issues where people delete one or both sets of backticks and break the formatting.

Maybe also add one empty line before and after the `Paste here` so that they're less inclined to delete the backticks.